src/commonMain/kotlin/exchange.dydx.abacus/processor/wallet/account/OrderProcessor.kt

```diff
@@ -202,6 +204,11 @@ internal class OrderProcessor(parser: ParserProtocol) : BaseProcessor(parser) {
             if (modified["id"] == null) {
                 modified.safeSet("id", payload["clientId"])
             }
+            parser.asInt(modified["subaccountNumber"])?.run {
+                modified.safeSet("subaccountNumber", this)
```

Comment:
personal preference, feel free to disregard if you disagree: i'm not a huge fan of using `.run { }` to pass `this` into things, since at a glance it can initially be ambiguous what `this` is.